### PR TITLE
PinkChain / Novo / Radiant / EthOne integration 

### PR DESCRIPTION
**Pull request by LoulouCrypto**
All test and working

### CODE REVIEW

File: src/Miningcore/Blockchain/Ethereum/EthereumPayoutHandler.cs [hunk 1]
```diff
@@ -386,22 +392,36 @@ private async Task<string> PayoutAsync(Balance balance, CancellationToken ct)
 
         var amount = (BigInteger) Math.Floor(balance.Amount * EthereumConstants.Wei);
 
-        var request = new SendTransactionRequest
+        SendTransactionRequest request = new SendTransactionRequest
```
Comment (oliverw) [inline src/Miningcore/Blockchain/Ethereum/EthereumPayoutHandler.cs:395]: Use var please

**Comment by oliverw** [comment]
Could you please add test cases for the new hash algos?

**Comment by LoulouCrypto** [comment]
> 

Can you tell me where to add id ? 

Thx

File: src/Miningcore/Blockchain/Ethereum/EthereumPayoutHandler.cs (hunk 1 above)
Comment (LoulouCrypto) [reply src/Miningcore/Blockchain/Ethereum/EthereumPayoutHandler.cs:395]: > Use var please

Will be done

**Comment by oliverw** [comment]
> > 
> 
> Can you tell me where to add id ?
> 
> Thx

https://github.com/oliverw/miningcore/blob/master/src/Miningcore.Tests/Crypto/HashingTests.cs

**Comment by oliverw** [comment]
Test is failing.

**Comment by LoulouCrypto** [comment]
> Test is failing.

Yes i see 
I'm digging

**Comment by LoulouCrypto** [comment]
> Test is failing.

I success ;) 

I think i made all modification needed .;) 

Hope it will be good and the community will appreciate ;)

File: src/Miningcore/Blockchain/Ethereum/EthereumPayoutHandler.cs (hunk 1 above)
Comment (LoulouCrypto) [reply src/Miningcore/Blockchain/Ethereum/EthereumPayoutHandler.cs]: Done

**Comment by minershaven** [comment]
Been using this build (with a few additions I made not related to his additions like adding minotaurx algo) and have had no problems or issues. RXD and NOVO finds blocks and pays out perfectly.  Nice work!

**Comment by oliverw** [comment]
Thanks a ton for your contribution. 👍

**Pull request merged**